AUTHORS

```diff
@@ -98,6 +98,7 @@ Dharmesh Poddar <dharmeshpoddar3@gmail.com>
 Diana Chen <dkchen1@asu.edu>
```

Comment:
I was expecting to see a version number change in package.json file but it seems like the pre-commit hooks are removing those changes (not sure!)

@vojtechjelinek, Can you please check this? (I'm assuming the package.json changes are not very important as of now so it should not block this PR.)

Reply:
This was fixed in #11365 but didn't got into the release.

Reply:
Ah, I see. Thanks for the info.

Reply:
Just as an FYI, I'm not going to cherry-pick that PR for the current release as we have already crossed that step.

Reply:
Yeah, sure, no problem, the package.json change is really marginal.